modules/inbucket/inbucket.go

```diff
@@ -0,0 +1,73 @@
+package inbucket
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+// InbucketContainer represents the Inbucket container type used in the module
+type InbucketContainer struct {
+	testcontainers.Container
+}
+
+// Smtp returns the connection string for the smtp server, using the default
+// 2500 port, and obtaining the host and exposed port from the container.
+func (c *InbucketContainer) Smtp(ctx context.Context) (string, error) {
```

Comment:
Wdyt about this name?
```suggestion
func (c *InbucketContainer) SmtpConnection(ctx context.Context) (string, error) {
```

Reply:
Done